build_defs/kotlin_native/kotlinc_wrapper.sh

```diff
@@ -1,83 +1,31 @@
 #!/bin/bash
 # Script to run the Kotlin Native Compiler.
-# Expects a comma-delimited list of dependency files as a first argument.
-# The rest of the arguments are passed in to the `kotlinc` compiler.
 
-
-# Create directory if it doesn't exist
-soft_mkdir() {
-  if [ ! -d "$1" ]; then
-   mkdir $1
-  fi
-}
-
-
-# Create file if it doesn't exist
-soft_touch() {
-  if [ ! -f "$1" ]; then
-    touch $1
-  fi
+abspath() {
+  echo "$(cd "$(dirname "$1")"; pwd -P)/$(basename "$1")"
 }
 
-_DEPENDENCIES=$1
-shift 1
-
-case $(ls "${BASH_SOURCE[0]}.runfiles/" | grep "kotlin_native_") in
+case $(ls "${BASH_SOURCE[0]}.runfiles/kotlin_native_local/kotlin-native-1.3.50/" | grep "kotlin-native-") in
```

Comment:
not a big deal, but is there an easy way to avoid hard coding the version and use the value in repo.bzl?

Reply:
I think I found a good way. It's also simpler.